docs/user/guides/patching_agama_live.md

```diff
@@ -0,0 +1,114 @@
+  one automatically) or adding your own modules.
+
+If you are familiar with Driver Update Disks, you might be wondering how to run your own scripts.
+The [Running scripts](#running-scripts) section explains how you do it.
+
+### Modifying the image
+
+You can modify the Agama live image using an RPM package or a Driver Update Disk. In the former
+case, the package is unpacked to the live image without running any script. If you prefer a Driver
+Update Disk, the files under the
+[inst-sys directory](https://ftp.suse.com/pub/people/hvogel/Update-Media-HOWTO/Update-Media-HOWTO.html#id_inst_sys)
+will be copied to the live image.
+
+### Adding packages for installation
+
+If you want to replace a package (or add a new one), you can place them under the `install`
+directory of your Driver Update Disk. Those packages will be copied to a special repository that
+Agama will use during installation. That repository will have higher priority than the product
+repositories, so those packages will be preferred.
+
+This purpose of this feature is replacing existing packages with updated or patched versions. If you
+want to add packages that are not in the distribution, you might need to tell Agama about those
+packages.
+
+### Adding or updating kernel modules
+
+The `modules` directory of your Driver Update might contain a set of kernel modules. Agama will copy
+them to the live image. If a `module.order` file is found, Agama will unload the modules listed in
+that file and it will make sure they are loaded later by systemd.
+
+## Running scripts
+
+Agama update mechanism will ignore the `update.*` scripts included in the Driver Update Disk. There
+are two reasons for that:
+
+1. Agama already offers a mechanism to run scripts.
+2. Most of those scripts will not work because the installation media has changed quite a lot.
+
+However, you can run scripts (and do many more things) if you include a partial Agama profile as
+part of the update. Place the file (`autoinst.jsonnet`, `autoinst.json` or `autoinst.xml`) in the
+`inst-sys` directory. It will be copied and automatically processed by Agama later.
+
+Please, refer to the
+[Starting the installation](/docs/overview/unattended#starting-the-installation) for further
+information.
+
+## Skipping security checks
+
+Although it is not recommended, you might want to skip security checks in some situations. You can
+use the following kernel arguments:
+
+- `inst.dud_insecure`: ignores SSL/TLS problems like a self-signed certificate when downloaqding the
+  update from an HTTPS server.
+- `inst.dud_packages.gpg`: enables/disables GPG signatures check when installing the packages
+  included in the DUD to the target system. The check is enable by default; set
+  `inst.dud_packages.gpg=0` to disable it.
+
+## Debugging
+
+If the update was not applied as you expected, you can switch to a console (e.g., pressing
+`alt+ctrl+f8`) and check the logs with the following command:
```

Comment:
very very NP: I'm a bit old school but we always used to say "ctr+al+" .... (del in case of windows ;-))